board/views.py

```diff
@@ -1,145 +1,32 @@
-import os
 import datetime
-from requests import post, get
 from django.shortcuts import render
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
+from gamestore.settings import BEARER_TOKEN, API_CLIENT_ID, API_SECRET_KEY
```

Comment:
A developer would not be able to tell which credentials are for which API by names used here.

Reply:
Also it's better to just import settings module and access the values from it.